cylc/flow/cfgspec/workflow.py

```diff
@@ -1995,6 +1973,25 @@ def upg(cfg, descr):
         ['cylc', 'mail', 'task event batch interval'],
         silent=cylc.flow.flags.cylc7_back_compat,
     )
+    u.deprecate(
+        '8.0.0',
+        ['runtime', '__MANY__', 'suite state polling'],
+        ['runtime', '__MANY__', 'workflow state polling'],
+        silent=cylc.flow.flags.cylc7_back_compat,
+        is_section=True,
+    )
+    for item in ('host', 'user'):
+        u.obsolete(
+            '8.0.0', ['runtime', '__MANY__', 'workflow state polling', item],
```

Comment:
Maybe not - host and user were not used in any version of Cylc 8.  So they were technically obsolete despite not been marked as such during parsing.

Reply:
At least `message -> output` below should be 8.3.0

Reply:
> Maybe not - host and user were not used in any version of Cylc 8. So they were technically obsolete despite not been marked as such during parsing.

Actually, worse than that - if host or user were configured, it would create a broken command line for the auto polling task.